.chloggen/add_missing_config_patter_purefa.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/purefareceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add missing scrapers to the config file
```

I believe this should be written as:

```suggestion
note: Updating documentation to include required configuration fields.
```

---

Sure @MovieStoreGuy thank you